esm/vitest.test.js

```diff
@@ -1,107 +1,112 @@
-
 import { describe } from "vitest";
```

Comment:
Please re-run the lint. This should have been fixed in the pre-commit hook. I'm not sure how this got through?